src/app/dashboard/Missions/DetailsText.jsx

```diff
@@ -0,0 +1,30 @@
+import { makeStyles } from "@material-ui/core/styles";
+import React, { useEffect, useState } from "react";
+import _ from "../../utils/lodash";
+
+const FoodBoxDetails = ({ showType, details }) => {
+  return (
+    <>
+      {showType ? <b>Food Box</b> : null}
+      {_.get(details, "needs")?.map((box, index) => {
```

Comment:
yeah, sorry, the coding standard change on this, lets go with details?.needs

Reply:
You got it.  Updated.